src/database_metrics/README.md

```diff
@@ -0,0 +1,17 @@
+# Capturing Database Metrics
+This module contains methods for collecting metrics on the database server under different conditions.
+
+## Usage
+1. Using the [Docker tutorials](https://github.com/cmu-delphi/delphi-epidata/blob/main/docs/epidata_development.md),
+install Docker and clone all the relevant repositories to a working directory. Do not build the images yet.
+2. Obtain the test data you will be running. Place it into a `common_full/`folder in the same working directory, with 
+sub folders corresponding to the production ingestion structure: `common_full/covidcast/receiving/<data_source>/filename`
+3. Before building the images, you will need to make two edits:  
+    a. Edit `repos/delphi/operations/src/secrets.py` by setting `db.host = 'delphi_database_epidata'` and 
+    `set db.epi = ('user', 'pass')`, which will match the testing docker image.  
+    b. Add `COPY common_full common_full` to `repos/delphi/operations/dev/docker/python/Dockerfile` after copying source files.
+4. Continue with the tutorial to complete the following steps:
+    a. Build the `delphi_web`, `delphi_web_epidata`, `delphi_database`, and `delphi_python` images according to the tutorial.  
+    b. Start the `delphi-net` network according to the tutorial.  
```

Comment:
small nit: you shouldn't have to do anything to start the network. did you maybe mean "create" instead of "start"?

Reply:
Good catch, edited to `create`